src/arch/xtensa/configs/acp_6_3_defconfig

```diff
@@ -34,3 +34,4 @@ CONFIG_COMP_TDFB=n
 #CONFIG_COMP_MUX=n
 CONFIG_COMP_SEL=n
 CONFIG_COMP_MIXER=n
+CONFIG_BINARY_BUILD=n
```

Comment:
Ths is a bit odd as there's no BINARY_BUILD Kconfig option defined in the tree. I guess this works,but is not the standard approach.

Reply:
nnno, let's not do this. All `CONFIG_*` symbols *must* have Kconfig entries. It becomes very confusing otherwise. Please add a Kconfig option or make it a generic macro, then of course you cannot define it in *_defconfig. Let me block this for clarification, I stand open for discussion of course